app/assets/stylesheets/application.scss

```diff
@@ -30,6 +30,56 @@
  //   -ms-transform: translateY(-50%);
  //   transform: translateY(-50%);
  // }
+ .dev-tag{
+  color: black;
+  width: 30px;
+  background-color: #A80874;
+  margin: 5px 5px 5px 5px;
+  border-width: 1px;
+  border-style: solid;
+  border-color: black;
+ }
+
+ .test-tag{
+  color: black;
+  width: 32px;
+  background-color: #B7FDFE;
+  margin: 5px 5px 5px 5px;
+  border-width: 1px;
+  border-style: solid;
+  border-color: black;
+ }
+
+ .qa-tag{
+  color: black;
+  width: 30px;
+  background-color: #2B9720;
```

Comment:
Color literals like `#2B9720` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#2B9720` should be written as `#2b9720`